client/src/components/edit-user-component/index.jsx

```diff
@@ -109,16 +126,32 @@ function EditComponent() {
           </CardContent>
```

Comment:
For some reason I can't highlight the textfield component above this, but remove the requirement for password to be specified. Users may not want to edit the password of an existing user -- allow an empty value to indicate no password change. I believe the backend is already built out to handle the case where the password is left empty.

Reply:
Sounds good, I removed the requirement.